### PR TITLE
✨ Add Recod Design for doctypes

### PR DESCRIPTION
**Pull request by AminovE99**
Print Formats for new Doctypes.

> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

recod_print_format_delivery_note
recod_print_format_sales_order
recod_print_format_purchase_order
recod_print_format_request_for_quotation

### CODE REVIEW

File: recod_erpnext_design/fixtures/print_format.json [hunk 1]
```diff
@@ -1,98 +1,194 @@
...
+  "html": null,
+  "line_breaks": 0,
+  "modified": "2020-08-13 07:37:53.307202",
+  "module": "Stock",
```
Comment (madmath03) [inline recod_erpnext_design/fixtures/print_format.json]: Wrong module.
```suggestion
  "module": "Recod ERPNext Design",
```

File: recod_erpnext_design/fixtures/print_format.json [hunk 2]
```diff
@@ -1,98 +1,194 @@
...
+  "html": null,
+  "line_breaks": 0,
+  "modified": "2020-08-13 07:41:48.380718",
+  "module": "Buying",
```
Comment (madmath03) [inline recod_erpnext_design/fixtures/print_format.json]: Wrong module.
```suggestion
  "module": "Recod ERPNext Design",
```

File: recod_erpnext_design/fixtures/print_format.json [hunk 3]
```diff
@@ -1,98 +1,194 @@
...
+  "html": null,
+  "line_breaks": 0,
+  "modified": "2020-08-13 07:43:48.019440",
+  "module": "Buying",
```
Comment (madmath03) [inline recod_erpnext_design/fixtures/print_format.json]: Wrong module.
```suggestion
  "module": "Recod ERPNext Design",
```

File: recod_erpnext_design/fixtures/print_format.json [hunk 4]
```diff
@@ -1,98 +1,194 @@
...
+  "html": null,
+  "line_breaks": 0,
+  "modified": "2020-08-13 07:48:03.530228",
+  "module": "Selling",
```
Comment (madmath03) [inline recod_erpnext_design/fixtures/print_format.json]: Wrong module.
```suggestion
  "module": "Recod ERPNext Design",
```

**Review by madmath03 — CHANGES_REQUESTED** [review]

**Comment by rai-monogramm** [comment]
Ok, you can implement them

**Review by madmath03 — APPROVED** [review]

**Pull request merged**